sdk/ai/azure-ai-client/samples/agents/sample_agents_basics.py

```diff
@@ -34,30 +35,34 @@
     conn_str=os.environ["AI_CLIENT_CONNECTION_STRING"],
 )
 
-with ai_client:
-    agent = ai_client.agents.create_agent(
-        model="gpt-4-1106-preview", name="my-assistant", instructions="You are helpful assistant"
-    )
-    print(f"Created agent, agent ID: {agent.id}")
+scenario = os.path.basename(__file__)
+tracer = configure_tracing("agent-samples").get_tracer(scenario)
 
-    thread = ai_client.agents.create_thread()
-    print(f"Created thread, thread ID: {thread.id}")
+with tracer.start_as_current_span(scenario):
```

Comment:
todo: add tracing-specific examples - this is temporary, just to see how tracing works in different scenarios